main.js

```diff
@@ -0,0 +1,166 @@
+class Book {
```

Comment:
- [ ] Use a function constructor instead of a class. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/book_array_requirement.png) from Odin)